BuggyProject/ViewController.swift

```diff
@@ -25,9 +25,23 @@ class ViewController: UIViewController {
 
     // MARK: Actions
     @IBAction func bTap() {
+        if let search = textField?.text {
```

Comment:
when you don't have an `else` clause, it's preferable to use a `guard`. does that make sense?